doc/examples/segmentation/plot_perimeters.py

```diff
@@ -33,25 +34,24 @@
 dX = X[0, 1] - X[0, 0]
 true_perimeters = [80 * scale, 2 * np.pi * R / dX]
 
-# for each type of objects, the different perimeters are evaluated
+# For each type of object, estimate its perimeter as the object is rotated,
+# according to different approximations
 for index, obj in enumerate([square, disk]):
 
-    # 2 neighborhood configurations for measure.perimeter
-    for n in [4, 6]:
+    # 4 or 8 neighbourhood can be used by the classic perimeter estimator
```

Comment:
neighbourhood -> neighborhood

Reply:
@alexdesiqueira I hesitated and I went for the non-US spelling just because it's the parameter name: https://github.com/scikit-image/scikit-image/blob/55dee6421c1411388eb299f70244c90d956140ef/skimage/measure/_regionprops_utils.py#L186
Should we start a deprecation cycle for this parameter?...

Reply:
@mkcor thanks for pointing that out. Yeah, I think a deprecation cycle would be good. Could you open a new issue based on that? I'll check the other places we would need to deprecate, also. We could take advantage of a new PR to change all of them at once.